components/product/ProductShelf.tsx

```diff
@@ -37,48 +45,69 @@ function ProductShelf({
     return null;
   }
 
+  const renderCarouselItems = () => {
+    const itemsToRender = products?.map((product, index) => (
+      <Slider.Item
+        index={index}
+        class="carousel-item min-w-[166px] w-[38.6vw] max-w-[380px] lg:w-[22.08vw] sm:first:pl-0 sm:last:pr-0"
+      >
+        <ProductCard
+          product={product}
+          itemListName={title}
+          layout={cardLayout}
+          platform={platform}
+          index={index}
+        />
+      </Slider.Item>
+    ));
+
+    if (carouselType === "See More Link") {
+      const lastProduct = products[products.length - 1];
+      itemsToRender[products.length - 1] = (
+        <Slider.Item
+          index={products.length - 1}
+          class="carousel-item min-w-[166px] w-[38.6vw] max-w-[380px] lg:w-[22.08vw] sm:first:pl-0 sm:last:pr-0"
+        >
+          <a class="w-full flex items-center justify-center" href={seeMoreLink}>
+            <span className="px-5 py-1.5 uppercase block mt-2 text-center border rounded-lg border-black">{seeMoreText}</span> 
+          </a>
+        </Slider.Item>
+      );
+    }
+
+    return itemsToRender;
+  };
+
   return (
-    <div class="w-full container  py-8 flex flex-col gap-12 lg:gap-16 lg:py-10">
+    <div class="w-full container max-w-[1630px] py-8 flex flex-col gap-5 lg:gap-6 lg:py-10">
       <Header
         title={title || ""}
-        description={description || ""}
-        fontSize={layout?.headerfontSize || "Large"}
-        alignment={layout?.headerAlignment || "center"}
+        fontSize={layout?.headerfontSize || "Normal"}
+        alignment={layout?.headerAlignment || "left"} 
       />
 
       <div
         id={id}
-        class="container grid grid-cols-[48px_1fr_48px] px-0 sm:px-5"
+        class="container max-w-[1630px] grid grid-cols-[48px_1fr_48px] px-4 sm:px-8"
       >
-        <Slider class="carousel carousel-center sm:carousel-end gap-6 col-span-full row-start-2 row-end-5">
-          {products?.map((product, index) => (
-            <Slider.Item
-              index={index}
-              class="carousel-item w-[270px] sm:w-[292px] first:pl-6 sm:first:pl-0 last:pr-6 sm:last:pr-0"
-            >
-              <ProductCard
-                product={product}
-                itemListName={title}
-                layout={cardLayout}
-                platform={platform}
-                index={index}
-              />
-            </Slider.Item>
-          ))}
+        <Slider class="carousel carousel-center sm:carousel-end gap-2 lg:gap-[15px] col-span-full row-start-2 row-end-5">
+          {renderCarouselItems()}  
```

Comment:
Uma função que retorna um JSX é um componente, então se atente a sintaxe aqui (se necessario até envie props):
```suggestion
          <CarouselItems />
```

Obs: Não acho que voce precisa disso aqui